.github/workflows/main.yml

```diff
@@ -101,9 +101,16 @@ jobs:
           server_path="test/conda_mock/static/packages"
           powerloader_path="build/powerloader"
           $powerloader_path download -f test/remote_mirrors.yml -d $server_path
+
+          # Upload file to AWS mock
           aws --endpoint-url $AWS_S3_ENDPOINT s3 cp $server_path/xtensor-0.23.9-hc021e02_1.tar.bz2 s3://testbucket/
           # aws --endpoint-url $AWS_S3_ENDPOINT s3 ls s3://testbucket
 
+          # Upload file to OCI mock: https://docs.docker.com/engine/reference/commandline/tag/
+          mkdir test/tmp
+          echo "artifact" > test/tmp/mock_artifact
+          oras push localhost:5000/mock_artifact:1.0 --manifest-config \
```

Comment:
I don't understand how this is supposed to work? Where does `oras` come from?

Reply:
In fact it looks like it's pre-installed. Nice! https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#cli-tools

Reply:
Good point, I already had it set up on my machine. Perhaps the line below would help?
https://github.com/mamba-org/powerloader/blob/25e005fd9712e2f2c59980303112d62767405fd5/.github/workflows/main.yml#L92

Reply:
no, we don't need it :)

Reply:
aah, because I can use powerloader to upload the artifact :)